src/components/CompareResults/RevisionRowExpandable.tsx

```diff
@@ -6,10 +6,15 @@ import { useAppSelector } from '../../hooks/app';
 import { Strings } from '../../resources/Strings';
 import { Colors, Spacing } from '../../styles';
 import type { CompareResultsItem } from '../../types/state';
+import { formatNumber } from './../../utils/format';
 
 const strings = Strings.components.expandableRow;
 const { singleRun } = strings;
-
+const NumberFormat = (value: number) => {
+  return new Intl.NumberFormat('en-US', {
+    maximumFractionDigits: 2,
```

Comment:
same here: the NumberFormat formatter needs to be instanciated only once

Reply:
After you create a formatter I think you won't need the function anymore, just call the formatter directly in the component.

Reply:
Updated now!